### ISSUE TITLE
Cache not PURGEing per page

### ISSUE THREAD

**Issue by jhubb-io** [thoughtfulweb/craft-litespeedcache#7]
Pages are not purging on save. We have {% cache %} tags on every page and the path to the lscache folder is set up properly.

**Comment by jsmrtn** [comment]
Are you running Craft 3.5, I wonder? Craft has deprecated template caches, so I assume the method of interaction that this plugin uses is now invalid.

We've moved away from using LSCache in favour of other static cache plugins, so considering this part would need a complete re-write, it's not likely to happen from our end any time soon.

**Comment by jhubb-io** [comment]
Yes, we are using Craft 3.5. I guess that you're not maintaining this plugin any more then? Just curious which static cache plugins you're now using?

**Comment by jsmrtn** [comment]
If it's something that's easily fixable it's likely to get patched i.e. [1.2.6](https://github.com/scaramangagency/craft-litespeedcache/releases/tag/1.2.6). As I mentioned, this would require a complete overhaul, so being we've no 'horse in the race' with the plugin, it's unlikely.

We use [Blitz](https://putyourlightson.com/plugins/blitz), now.
